pipeline/pipelines/code_generation_pipeline.py

```diff
@@ -103,10 +116,16 @@ def do_build_flow(self, **kwargs):
                  format_tasks.PythonFormatTask('PythonFormat', inject=kwargs),
                  gapic_tasks.GapicMergeTask('GapicMerge', inject=kwargs),
                  gapic_tasks.GapicPackmanTask('GapicPackman', inject=kwargs))
+        if kwargs['publish_env'] != "none":
```

Comment:
changed to single quotes.  also changed to desired behavior (based on presence of publish_env) as mentioned by jmuk